src/entities/team.ts

```diff
@@ -17,11 +18,11 @@ export class Team extends BaseEntity {
   @Column()
   name: string;
 
-  @Column()
-  location: string;
+  @Column({ unique: true })
+  tableNumber: string;
```

Comment:
Should this be a number?

Reply:
I check if it's a number when I collect it, but I thought maybe a number might be less brittle... thoughts?

Reply:
Well if we want to make sure it's a number, I would prefer that we save it as a number in the database. Easier to migrate `number -> string` than `string -> number`